src/openrct2/scripting/bindings/entity/ScGuest.cpp

```diff
@@ -507,6 +508,22 @@ namespace OpenRCT2::Scripting
         }
     }
 
+    uint16_t ScGuest::favouriteRide_get() const
+    {
+        auto peep = GetGuest();
+        return peep != nullptr ? peep->FavouriteRide.ToUnderlying() : RideId::GetNull().ToUnderlying();
+    }
+
+    void ScGuest::favouriteRide_set(uint16_t value)
+    {
+        ThrowIfGameStateNotMutable();
+        auto peep = GetGuest();
+        if (peep != nullptr)
+        {
+            peep->FavouriteRide = RideId::FromUnderlying(value);
```

Comment:
Similarly, IMHO the favourite ride should reset when `null` is assigned.

Reply:
Should I use `DukValue` for function return & param type then? That seems to be what similar properties do.

Reply:
Yes. 🙂